packages/meditrak-server/src/apiV2/middleware/emailAfterTimeout.js

```diff
@@ -31,9 +31,9 @@ const setupEmailResponse = async (req, res, constructEmailFromResponse) => {
 
   // override the respond function so that when the endpoint handler finishes (or throws an error),
   // the response is sent via email
-  res.overrideRespond = responseBody => {
-    const { subject, message } = constructEmailFromResponse(responseBody);
-    sendResponseAsEmail(user, subject, message);
+  res.overrideRespond = async responseBody => {
```

Comment:
Since we don't `await` for `sendResponseAsEmail()` in this function:

```suggestion
  res.overrideRespond = responseBody => {
```